content/en/blog/2023-03-01-february-update/index.md

```diff
@@ -0,0 +1,259 @@
+
+### Next Flux release: more stability and performance improvements
+
+We have released [Flux v0.39](https://github.com/fluxcd/flux2/releases/tag/v0.39.0).
+Users are encouraged to upgrade for the best experience.
+
+Highlights
+
+- Starting with this version, the Flux controllers come with [SBOMs and
+  SLSA Provenance Attestations](/flux/security/)
+  embedded in their container images.
+- The [Flux Terraform Provider](https://github.com/fluxcd/terraform-provider-flux)
+  has a new resource for bootstrapping Flux, without depending on
+  third-party Terraform providers, that allows customising the
+  controllers at install time. Users are encouraged to migrate to
+  this new resource and provide feedback.
+- The Flux CLI is now included in [Wolfi OS](https://github.com/wolfi-dev/os),
+  the Linux (Un)distro designed for securing the software supply chain. The
+  Chainguard team and Wolfi maintainers are shipping updates for the Flux
+  package on a regular basis.
+
+Features and improvements
+
+- Recreate immutable resources (e.g. Kubernetes Jobs) by annotating
+  or labeling them with `kustomize.toolkit.fluxcd.io/force: enabled`.
+- Support for HTTPS bearer token authentication for Git repositories.
+- Improve memory usage by disabling the caching of `Secret` and
+  `ConfigMap` resources in all controllers.
+- Better observability with progressive status updates for Sources
+  (Git, OCI, Helm, S3 Buckets).
+- Allow extracting the OCI artifact SHA256 digest for Cosign with
+  `flux push artifact -o json`.
+- Track CRDs managed by Flux, `flux trace` and `flux tree` will show
+  which HelmRelease deployed which CRDs.
+- Allow the Flux GitHub Action to use a GitHub token when checking
+  for updates to avoid rate limiting.
+
+Documentation
+
+- Security: [Software Bill of Materials](/flux/security/#software-bill-of-materials)
+- Security: [SLSA Provenance Attestations](/flux/security/#slsa-provenance-attestations)
+- Security: [Scanning Flux images for CVEs](/flux/security/#scanning-for-cves)
+
+Big thanks to all the Flux contributors that helped us with this release!
+
+### Security news
+
+### Flagger x.y.z
+
+### Flux Ecosystem
+
+<!--
+
+If you add entries to this subsection, please don't use "we" as it gets
+confusing from which perspective the whole blog post is written and who
+"we" is. The whole post is meant to be from the perspective of the Flux
+community. Better to write:]
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
+#### Terraform-controller
+
+#### Weave GitOps
+
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.17.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.16.0>
+
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
+#### New additions to the Flux Ecosystem
+
+## Recent & Upcoming Events
+
```

Comment:
@vanessaabankwah @staceypotter What shall we put in here for Jan/Feb/Mar?

Reply:
[2nd March: GitOps Testing in Kubernetes with Flux & Testkube](https://www.meetup.com/gitops-community/events/291670250?utm_medium=referral&utm_campaign=share-btn_savedevents_share_modal&utm_source=link) 

[9th March: Microservices and Kubernetes](https://community.cncf.io/events/details/cncf-cncf-online-programs-presents-cncf-on-demand-webinar-the-path-to-cloud-adoption-and-app-modernization/)

[15th March: CNCF Live Stream -Automating Kubernetes Deployments](https://community.cncf.io/e/mbmpq8/)

[16th March: CNCF On Demand- Kubernetes in 2023 w/ Stefan Prodan & Brendan Burns](https://community.cncf.io/events/details/cncf-cncf-online-programs-presents-cncf-on-demand-webinar-kubernetes-in-2023/)

23rd March: Microsoft Live Stream: Automating Kubernetes Deployments

Reply:
Sweet!